packages/client/src/views/UserAudit/UserAuditHistory.tsx

```diff
@@ -325,7 +325,11 @@ class UserAuditHistoryComponent extends React.Component<Props, State> {
         trackingIdString: isUserAuditItemWithDeclarationDetials(userAuditItem)
           ? userAuditItem.data.trackingId
           : null,
-        auditTime: format(new Date(userAuditItem.time), 'MMMM dd, yyyy hh:mm a')
+        auditTime: format(
+          new Date(userAuditItem.time),
+          'MMMM dd, yyyy hh:mm a'
+        ),
+        auditTime24: format(new Date(userAuditItem.time), 'MMMM dd, yyyy HH:mm')
```

Comment:
```suggestion
        auditTime24: new Date(userAuditItem.time)
```
This should make it simpler right?

Reply:
new Date(userAuditItem.time) returns string
(new Date(userAuditItem.time)).getTime() returns timestamp

Reply:
We seems to be using `orderBy` function from lodash. I'm guessing event if we use only `new Date(userAuditItem.time)` the sorting should still work. Could you take a look and confirm please? BTW instead of `auditTime24`, lets use `auditTimeValue`

Reply:
yes, the sorting works correctly with `new Date(userAuditItem.time)` .

Reply:
ok cool!